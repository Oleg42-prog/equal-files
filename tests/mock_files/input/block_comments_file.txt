Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
Sed dictum, elit a luctus lacinia, velit nibh laoreet nunc, non pretium nunc odio vel justo. /**/
/*
hiiiii
*/
Sed vel tellus nec nisi posuere cursus. 
Nulla facilisi. /*

some body was told me

*/
Sed euismod, nunc non bibendum viverra, justo nisl tristique risus, nec tincidunt nisl nunc quis nulla. 
/**/
/*
wrum
    /*
    wroom
        /*
            wrooom
        */
    */
*/
Integer nec nulla vel nunc elementum lacinia. /* the world is gonna roll me */
/* Calm down */Sed euismod, nunc non bibendum viverra, justo nisl tristique risus, nec tincidunt nisl nunc quis nulla. 
Phasellus consequat, nisl eu gravida tincidunt, velit orci auctor nulla, vitae aliquam nulla quam nec nulla.
