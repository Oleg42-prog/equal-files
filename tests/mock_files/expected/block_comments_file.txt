Lorem ipsum dolor sit amet, consectetur adipiscing elit. 
Sed dictum, elit a luctus lacinia, velit nibh laoreet nunc, non pretium nunc odio vel justo. 

Sed vel tellus nec nisi posuere cursus. 
Nulla facilisi. 
Sed euismod, nunc non bibendum viverra, justo nisl tristique risus, nec tincidunt nisl nunc quis nulla. 


Integer nec nulla vel nunc elementum lacinia. 
Sed euismod, nunc non bibendum viverra, justo nisl tristique risus, nec tincidunt nisl nunc quis nulla. 
Phasellus consequat, nisl eu gravida tincidunt, velit orci auctor nulla, vitae aliquam nulla quam nec nulla.
